packages/components/src/components/data-list/data-list.tsx

```diff
@@ -57,14 +60,19 @@ const DataList = React.memo(
     ({
         children,
         className,
+        clearScroll = () => {
+            // do nothing
+        },
```

Comment:
since its already optional above, i think we need to define it here

Reply:
i think it would be better to give a default value instead of leaving it undefined.